bin/lib/game.js

```diff
@@ -48,6 +48,16 @@ if(process.env.GAME_VARIANT !== undefined){
 	}
 }
 
+const GAME_DISTANCE_OF_WRONG1 = ( process.env.GAME_DISTANCE_OF_WRONG1 !== undefined )? parseInt(process.env.GAME_DISTANCE_OF_WRONG1) : 2;
+const GAME_DISTANCE_OF_WRONG2 = ( process.env.GAME_DISTANCE_OF_WRONG2 !== undefined )? parseInt(process.env.GAME_DISTANCE_OF_WRONG2) : 3;
+
+if (GAME_DISTANCE_OF_WRONG1 < 1) {
+	throw `ERROR: process.env.GAME_DISTANCE_OF_WRONG1(${process.env.GAME_DISTANCE_OF_WRONG1}) should be >= 1`
+}
+if (GAME_DISTANCE_OF_WRONG2 < 1) {
+	throw `ERROR: process.env.GAME_DISTANCE_OF_WRONG2(${process.env.GAME_DISTANCE_OF_WRONG2}) should be >= 1`
+}
+
```

Comment:
What happens if `GAME_DISTANCE_OF_WRONG2` is less than `GAME_DISTANCE_OF_WRONG1`?